internal/kubecmd/kubecmd.go

```diff
@@ -393,6 +399,39 @@ func (k *KubeCmd) applyNodeVersion(ctx context.Context, nodeVersion updatev1alph
 	return updatedNodeVersion, err
 }
 
+func (k *KubeCmd) reconcileKubeadmConfigMap(ctx context.Context) error {
```

Comment:
We should save the kubeconfig to disk after running this function, otherwise we will have to keep it for as long as there are clusters from before v2.14 still running.

Reply:
I don't fully understand your point. I'm changing the Kubeadm ConfigMap here. We can remove this function after we've released v2.14 since our upgrade path states that the user must go through every Constellation minor release.

Reply:
Ahhh, I thought we were updating the config of the user, not the config present in the cluster.
In that case its fine